test/invokeArgs.js

```diff
@@ -0,0 +1,52 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('invokeArgs', function () {
+  const testObj = {
+    innerObj: {
+      method(a, b, c) {
+        return a + b + c;
+      },
+    },
+    method(a, b, c) {
+      return a + b + c;
+    },
+    methodWithoutArgs() {
+      return 1;
+    },
+    notAMethod: 'Hello',
+  };
+
+  const testFunc = (a, b) => a + b;
+
+  it('should invoke the method at path of object with given arguments', function () {
+    assert.strictEqual(RA.invokeArgs(['abs'], [-1], Math), 1);
+    assert.strictEqual(RA.invokeArgs(['method'], [1, 2, 3], testObj), 6);
+    assert.strictEqual(
+      RA.invokeArgs(['innerObj', 'method'], [1, 2, 3], testObj),
+      6
+    );
+    assert.strictEqual(RA.invokeArgs([], [1, 2], testFunc), 3);
```

Comment:
This test should assert for `undefined` as expected value. As pseudocode suggest, it's a corner case that we have to take care explicitly to be compatible with `lodash`, `underscore` or any other invoke library.

We also talked about it [here](https://github.com/char0n/ramda-adjunct/pull/1409#discussion_r411964700).

Reply:
OK, I think I finally got this part